src/plugins/forceOwnerCrown/index.ts

```diff
@@ -29,8 +29,9 @@ export default definePlugin({
         {
             find: "#{intl::GUILD_OWNER}),children:",
             replacement: {
-                match: /,isOwner:(\i),/,
-                replace: ",_isOwner:$1=$self.isGuildOwner(e),"
+                // The isOwner prop is used in more tha one component in this module. Make sure we patch the right one
+                match: /,isOwner:(\i),(?=[^}]+guildId)/,
+                replace: ",_isOwner:$1=$self.isGuildOwner(arguments[0]),"
```

Comment:
maybe we can do something like this instead

```suggestion
                match: /(?<=decorators:.{0,200}?isOwner:)\i/,
                replace: "$self.isGuildOwner(arguments[0])"
```

Reply:
yeah, works fine.